plugins/org.eclipse.elk.alg.packing.rectangles/src/org/eclipse/elk/alg/packing/rectangles/firstiteration/FirstIteration.java

```diff
@@ -0,0 +1,341 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Kiel University and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Kiel University - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.elk.alg.packing.rectangles.firstiteration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.elk.alg.packing.rectangles.util.DrawingData;
+import org.eclipse.elk.alg.packing.rectangles.util.PackingStrategy;
+import org.eclipse.elk.alg.packing.rectangles.util.DrawingDataDescriptor;
+import org.eclipse.elk.graph.ElkNode;
+
+/**
+ * Class that handles the first iteration of the algorithm, producing an approximated bounding box for the packing.
+ * 
+ * @author dalu
+ */
+public class FirstIteration {
```

Comment:
Are there more descriptive names than `FirstIteration` and `SecondIteration`? Something that tells me what the steps actually do?

Reply:
More descriptive names will be added, including further descriptions in comments and the melk files.